docs/source/clusters/kaizen/Northeastern-Cluster.md

```diff
@@ -11,15 +11,15 @@ Instructions for [Accessing Northeastern Cluster](Accessing-Northeastern-Cluster
 [NEU Equipment Sign-Out](NEU-Equipment-Sign-Out.html)
 
   **SERVERS**
-  * Cisco UCS C220 M3 SSF (x48). See [this file](_static/pdf/CiscoConfiguration.pdf) for list of configuration components. 
+  * Cisco UCS C220 M3 SSF (x48). See [this file](../../_static/pdf/CiscoConfiguration.pdf) for list of configuration components. 
```

Comment:
Do these links work correctly?  I ran into some issues trying to use `..` in links when I was playing with things yesterday.  The `..` seemed to be ignored.

Reply:
Confirmed, this seems broken when I build it.  I think this needs to be `/_static/pdf/CiscoConfiguration.pdf`.

Reply:
We can add a check for `..` in link paths to our validation script!

Reply:
That's because we were using recommonmark v 0.4 instead of 0.5. https://www.sphinx-doc.org/en/master/usage/markdown.html Fixed that in the previous PR. It was ignoring ../, ./ and still resolved (Accessing-Northeastern-Cluster.html) to clusters/kaizen/clusters/kaizen/Accessing-Northeastern-Cluster.html

Here's what the page looks like with recommonmark 0.5 and ../../_static as the image path https://documentation-hgibson-fork.readthedocs.io/en/latest/clusters/kaizen/Northeastern-Cluster.html

Reply:
Oh, neat.  So we have two solutions :).  The question becomes: do we want to use relative links like `../..`, or should we just use absolute links (`/_static/...`) for things outside of the current directory?

Reply:
Thinking about it: The problem with relative links is that they break whenever you move a document. Absolute links wouldn't have that problem.

Reply:
Absolute links would mean fewer links would need to be changed if the file was moved, but you mentioned at the ops meeting that people might be able to understand relative links more intuitively.

Reply:
> The problem with relative links is that they break whenever you move a document. Absolute links wouldn't have that problem.

yeah, absolute path seem better here.

Reply:
If I said that I take it back upon reconsideration.

Reply:
Okay so absolute it is then?

Reply:
Just *update* the PR. No need to close it and open a new one.

Reply:
(Amend the existing commit, then `git push -f` to your branch on github)